src/screens/MapPage/MapPage.js

```diff
@@ -1,14 +1,32 @@
 import React from 'react'
-import { Box, Text } from 'grommet'
+import { Box } from 'grommet'
 import SidePanel from './components/SidePanel'
+import BaseMap from 'components/BaseMap'
+import styled from 'styled-components'
+import DrawingOverlay from 'components/DrawingOverlay'
+
+const Relative = styled.div`
+    position: relative;
+    width: 100%;
+`
 
 export default function MapPage() {
+  const [canDraw, changeDrawing] = React.useState(false)
+
   return (
     <Box direction='row' height={{ min: '100%' }}>
```

Comment:
Non-blocking: you can use `fill` to specify filling to the max width and height of the container: https://v2.grommet.io/box#fill

Reply:
I've found issues in the past using `fill`, which I've written up in a [previous PR](https://github.com/zooniverse/mapping-viz-tools/pull/4#issuecomment-662089127).